python/rikai/spark/sql/codegen/pytorch.py

```diff
@@ -25,6 +25,7 @@
 
 DEFAULT_NUM_WORKERS = 8
 DEFAULT_BATCH_SIZE = 4
+DEFAULT_PIN_MEMORY = True
```

Comment:
pytorch has pin memory defaulting to False because in some instances it can result in swap/freeze that's hard to diagnose